lib/hunter/commands/hunt.rb

```diff
@@ -50,6 +50,7 @@ def run
         @auto_apply ||= Config.auto_apply
 
         raise "No port provided!" if !@port
+        raise "Provided port #{@port} is busy" if !`lsof -i:#{@port}`.empty?
```

Comment:
What is the error that's given when trying to start a `hunt` process over a busy port? It may be a better idea to `rescue` the error; using `lsof` assumes that the system has `lsof` installed, which means adding it as a project dependency.

Reply:
It throws `Errno::EADDRINUSE` for lines 93 and 120 of `hunt.rb`, namely these lines:

`server.bind('0.0.0.0', @port)`
`server = TCPServer.open(@port)`

I considered rescuing the error but I felt like it was nicer to catch stuff like this in the main process instead of the child processes. There doesn't seem to be an easy way for Ruby to simply "check" that a port is busy, beyond trying to use it and seeing if that works. Up to you whether you think it's better to do that or to catch the error in each child thread.

Reply:
It's probably fine to leave it as is for now. I'll make sure to add `lsof` as a dependency when we tag a new version.